cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -37,12 +41,33 @@ def init_second_jwks_file_with_same_kid(first_file_name, second_file_name)
     )
   end
 
+  def init_ecdsa_jwks_file(file_name, key_type)
+    @default_file_name ||= file_name
+    @key_type = key_type
```

Comment:
maybe use  memoization here
```suggestion
    @key_type ||= key_type
```